src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -2551,4 +2591,20 @@ public void addAlias(String alias, String definition) {
       }
     }
   }
+
+  /**
+   * An environment specifies the built-in externs that are loaded for a given
+   * compilation.
+   */
+  public static enum Environment {
+    /**
+     * Hand crafted externs that have traditionally been the default externs.
+     */
+    LEGACY,
```

Comment:
Is this the same set of externs someone would want in the browser enviroment with es6? If so, we can pick some name related to that.